config/jobs/kubernetes/sig-node/containerd.yaml

```diff
@@ -339,11 +339,11 @@ periodics:
       args:
       - --root=/go/src
       - --repo=k8s.io/kubernetes=release-1.25
-      - --repo=github.com/containerd/containerd=release/1.5
+      - --repo=github.com/containerd/containerd=release/1.6
```

Comment:
see above comment... 1.25 is tied to 1.5 containerd only because 1.5 came out before cri v1 api was mandated in 1.26 (which was k8s master last time around and is now service)  So probably easier to just delete the 1.5 job